lighthouse-core/gather/gatherers/scripts.js

```diff
@@ -20,7 +22,7 @@ class Scripts extends Gatherer {
   async afterPass(passContext, loadData) {
     const driver = passContext.driver;
 
-    /** @type {Object<string, string>} */
+    /** @type {LH.Artifacts['Scripts']} */
```

Comment:
😶

Reply:
I think we should do `Array<LH.Artifacts['Scripts'][0]>` 😆

Reply:
😱 

counter arg: you know it's an array in fewer characters when you see `[]`